spec/integration/fields/multiple_carrierwave_spec.rb

```diff
@@ -30,7 +30,7 @@
 
     it 'supports deleting a file', js: true do
       visit edit_path(model_name: 'field_test', id: field_test.id)
-      click_link 'Delete carrierwave assets #1'
+      click_link I18n.t('admin.actions.delete.link_with_number', object_label: 'Carrierwave assets', object_number: 1)
```

Comment:
Can we restore this back to the original state? In this way we lose a way to check the content of I18n translation...